src/app/tests/suites/certification/Test_TC_CC_3_3.yaml

```diff
@@ -39,14 +39,14 @@ tests:
       response:
           value: 1
 
-    - label:
-          "Check current hue attribute value matched the value sent by the last
-          command"
-      disabled: true
+    - label: "Reads CurrentHue attribute from DUT"
```

Comment:
What is the point of this step, if we don't do anything with the value we read?

Can you please point me to the test plan in question?

Reply:
![cc-3-3](https://user-images.githubusercontent.com/57476670/166867080-5027a66b-1d8e-4e66-97c5-34ff3e82daed.png)

@bzbarsky-apple above step is current hue attribute  value range check as per Testplan